src/pages/Home.tsx

```diff
@@ -0,0 +1,74 @@
+import { useEffect } from "react";
+import { Button } from "../components/Button";
+import { Photo } from "../components/Photo";
+import { useFetchImage } from "../utils/api";
+import { Overlay } from "../style";
+import { withAction } from "../store";
+import { ActionState } from "../types";
+
+export const Home = withAction(
+  ({ appendAction }: { appendAction: Function }) => {
```

Comment:
Explicitly declare the params (use interface) being accepted here and in the withAction function.
`const HomeInner: React.FC<HomeActionProp> = ({ appendAction}) => {}`

Tip: Separate declaration of React Function Component and its usage in different lines for code clarity
`export const Home = withAction(HomeInner)`